source/_docs/automation/yaml.markdown

```diff
@@ -84,6 +84,28 @@ max_exceeded:
   required: false
   type: string
   default: warning
+trigger:
+  description: The trigger(s) which will start the automation. Multiple triggers can be added and the automation will start when any of these triggers changes from `false` to `true`.
```

Comment:
Triggers are not conditional (e.g., can be event based too); so its not about being true or false.

```suggestion
  description: The trigger(s) which will start the automation. Multiple triggers can be added and the automation will start when any of these triggers trigger.
```